content/en/serverless/aws_lambda/configuration.md

```diff
@@ -39,12 +39,12 @@ First, [install][1] Datadog Serverless Monitoring to begin collecting metrics, t
 - [Collect logs from non-Lambda resources](#collect-logs-from-non-lambda-resources)
 - [Collect traces from non-Lambda resources](#collect-traces-from-non-lambda-resources)
 - [Filter or scrub information from logs](#filter-or-scrub-information-from-logs)
-- [Disable logs collection](#disable-logs-collection)
+- [Enable/disable logs collection](#enabledisable-log-collection)
```

Comment:
```suggestion
```

Reply:
Is it expected for this link to be duplicated in the APM section?

Reply:
i seem to remember we wanted it to be linked in both places, so let's keep it in both for now.